libraries/AP_Mount/AP_Mount_Backend.h

```diff
@@ -64,10 +64,10 @@ class AP_Mount_Backend
     // control - control the mount
     virtual void control(int32_t pitch_or_lat, int32_t roll_or_lon, int32_t yaw_or_alt, MAV_MOUNT_MODE mount_mode);
     
-    // process MOUNT_CONFIGURE messages received from GCS:
+    // process MOUNT_CONFIGURE messages received from GCS. deprecated.
```

Comment:
Which code in ArduPilot is this deprecated in favour of?

Reply:
I have just copied the comment from the .cpp to make them consistent. not introducing anything new here

Reply:
these ardupilot specific messages had been deprecated btw long long time ago (several years), in favor of the COMMAND commands

(which btw introduced a serious bug I mentioned repeatedly but which nobody seems to care about lol, but that's another story lol)

Reply:
> these ardupilot specific messages had been deprecated btw long long time ago (several years), in favor of the COMMAND commands

Right.  I wonder if the GCSs actually *are* using the commands now.  If we can show that they are we can at least move to make these optionally-compiled.  Sadly, a quick grep in the MissionPlanner and MAVProxy source code shows these entry points still being used, which means we can't make these go away.
 
> (which btw introduced a serious bug I mentioned repeatedly but which nobody seems to care about lol, but that's another story lol)

That's the factor-of-100 bug?

Reply:
yes, still found in code, and also solo is using them AFAIK
yes

Reply:
> yes, still found in code, and also solo is using them AFAIK
> yes

IIRC we couldn't fix that without breaking things for people who updated their autopilot but not their GCS or vice-versa.  I guess at this point we're stuck with that bug until we transition to a newer command set.

If you can think of a way to get from here-to-there, please do mention it :-)

Reply:
well, that's a bit funny since some years back when it was introduced it must have been exactly the other way around ... but I agree that's not helpful
no, I do not have an idea for how to do the transition without affecting anyone 
my issue is also not so much that there is this bug, but that it is kind of not acknowledged, i.e., I think what one should at least do is to document it somehow so that it just is known (I recall a case some time ago there a GCS coder known to you spent time into investigating what's going on => wasted time), could be just a comment in the code next to the message handlers or so, or one even could put a comment into the mavlink description, e.g. in the command description or like it is done with the parameters, there would be plenty of ways to handle the situation more gracefully
if it would be made "official" that it is how it is (and that it's not going to change) one could deal with it - like we all learned to deal with the parameter difference. In that sense it wouldn't be a bug anymore but a known different behavior.